chrome/content/zotero/xpcom/zotero.js

```diff
@@ -2752,6 +2752,7 @@ Zotero.Browser = new function() {
 	
 	this.createHiddenBrowser = createHiddenBrowser;
 	this.deleteHiddenBrowser = deleteHiddenBrowser;
+	this.createHiddenBrowserWithURL = createHiddenBrowserWithURL;
```

Comment:
We don't do this anymore for methods. `this.createHiddenBrowserWithURL = function () {` below.